src/Sylius/Bundle/AttributeBundle/Controller/AttributeController.php

```diff
@@ -48,6 +48,8 @@ public function getAttributeTypesAction(Request $request, $template)
      */
     public function renderAttributesAction(Request $request)
     {
+        $template = $request->attributes->get('template', 'SyliusAttributeBundle::attributeChoice.html.twig');
```

Comment:
I will rework this routing and templates anyway, so let's not bother with that right now.